lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUCloseTaskTest.cs

```diff
@@ -53,7 +53,7 @@ public void TestTaskCloseOnLocalRuntime()
         {
             const int chunkSize = 2;
             const int dims = 50;
-            const int iterations = 200;
+            const int iterations = 1000;
```

Comment:
The purpose of the test is to wait for all the tasks are running then close them. If the iterations are too quick, by the time we receive the last running task, the first task can complete. Although it doesn't impact the test result as we count both completed and failed tasks, we would still like all the tasks are completed by close events rather than finish by itself. 
We are using IMRU tasks in test, there is no way to change the task to wait for a signal. 
And this iteration won't increase task execution time because as soon as we receive all the running tasks, we start to close tasks.